content/pt-br/gitops.md

```diff
@@ -0,0 +1,16 @@
+---
+title: GitOps
+status: Feedback Appreciated
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+GitOps é um conjunto de melhores práticas baseadas em [princípios compartilhados](https://opengitops.dev/), aplicadas a um fluxo de trabalho que depende de agentes de software que permitem a automação reconciliar um estado ou configuração declarada do sistema em um repositório git. Esses agentes de software e práticas são usados para executar um fluxo de trabalho coeso que utiliza um sistema de controle de versão como o Git como a "única fonte de verdade" e estende essa prática para aplicações, infraestrutura e procedimentos operacionais.
```

Comment:
What do you think of "controle de versão, como o Git, sendo a" instead of "controle de versão como o Git como a"? Looks better to me.

Reply:
Great! 😃